docs/user-documentation/context.md

```diff
@@ -40,6 +40,13 @@ Now, when a user creates a Media and tags it as a "Preservation Master File", th
 
 ![Context - thank user for preservation file: dsm](../assets/context_thanks_dsm.png)
 
+### Tips on Using Islandora-specific Contexts
+
+Islandora provides some Conditions and Reactions that aren't available on other Drupal sites. When using them, you may find the following tips useful:
+
+* When using Islandora's "Node has term with URI" and "Media has term with URI" Conditions, you may want to enter multiple terms to take advantage of the AND/OR logic they provide. To do so, add a comma (`,`) after the first term in the autocomplete field and continue typing.
```

Comment:
There's a third condition, "Parent node for media has term with URI", for which all this text is applicable as well.

Reply:
OK, added.